src/app/components/normal-component/header/header.component.html

```diff
@@ -13,6 +13,7 @@
             <span aria-hidden="true"></span>
             <span aria-hidden="true"></span>
             <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
```

Comment:
QUE QUE QUOITTE IS DAS CODE :D ??????

Reply:
c'est le hamburger menu quand t'es en mode téléphone, ils sont passé de "il faut trois span pour l'afficher" à "il faut 4 span pour afficher les 3 barres", sans cet ajout on se retrouve avec un hamburger à 2 barres